v22.2/security-reference/authorization.md

```diff
@@ -119,15 +136,43 @@ Roles and users can be granted the following privileges:
 
 {% include {{ page.version.version }}/sql/privileges.md %}
 
+### System-level privileges 
+
+<span class="version-tag">New in v22.2:</span> System-level privileges offer more granular control over a user's actions when working with CockroachDB, over the legacy [role options authorization model](#role-options).
```

Comment:
The two clauses in this sentence read a little disconnected — possibly because of the doubling up of "over" with two slightly different meanings. Maybe, slightly rephrase to something like:

System-level privileges offer more granular control over a user's actions when working with CockroachDB, compared to the legacy role options authorization model.